src/sentry/integrations/bitbucket/client.py

```diff
@@ -26,6 +26,7 @@ class BitbucketAPIPath:
 
     repository = "/2.0/repositories/{repo}"
     repositories = "/2.0/repositories/{username}"
+    repository_commit = "/2.0/repositories/{repo}/commit/{sha}"
```

Comment:
Are the BitBucket endpoints used anywhere?

Reply:
Not at the moment, but I plan to followup with one